data_test.go

```diff
@@ -1258,6 +1258,7 @@ func (d *DB) waitTableStats() {
 
 func runIngestAndExciseCmd(td *datadriven.TestData, d *DB, fs vfs.FS) error {
 	var exciseSpan KeyRange
+	var doFlushableIngest bool
```

Comment:
We should rename this and the `case` below to `sstContainsExciseTombstone` and `contains-excise-tombstone` to improve readability between the datadriven testdata and the IngestAndExcise code itself.